aspnetcore/signalr/authn-and-authz.md

```diff
@@ -103,6 +95,25 @@ var connection = new HubConnectionBuilder()
 
 Windows Authentication is only supported by the browser client when using Microsoft Internet Explorer or Microsoft Edge.
 
+### Using Claims to Customize Identity Handling
+
+If you're using organizational or social authentication in your apps and you'd prefer to have more fine-grained control over how SignalR identifies users, you can implement `IUserIdProvider` to return values from one of the claims collected by the identity provider. 
+
+The code below demonstrates how you would use Claims to select the user's email address as the identifying property. 
```

Comment:
The following code 
("below" is frowned upon because it doesn't make sense when read by screen readers)

demonstrates how ~~you would use Claims ~~ to select 

(if you keep the ref to claims here:  Claims --> claims)

Reply:
good feedback, thanks for that. i wasn't aware of those nuances but thanks for bringing them to my attention.